cluster/hooks.go

```diff
@@ -415,7 +415,8 @@ func InstallHorizontalPodAutoscalerPostHook(cluster CommonCluster) error {
 		return err
 	}
 
-	return installDeployment(cluster, infraNamespace, pkgHelm.BanzaiRepository+"/hpa-operator", "hpa-operator", valuesOverride, "", false)
+	return installDeployment(cluster, infraNamespace, pkgHelm.BanzaiRepository+"/hpa-operator",
+		"hpa-operator", valuesOverride, viper.GetString(pipConfig.AutoscaleHpaOperatorChartVersion), false)
```

Comment:
Move `viper.GetString(pipConfig.AutoscaleHpaOperatorChartVersion)` to the top where the others are.
```suggestion
		"hpa-operator", valuesOverride, chartVersion, false)
```

Reply:
changed